src/components/panels/watches.tsx

```diff
@@ -295,14 +295,14 @@ export function WatchesPanel(props: PanelProps<PlaygroundPanelLocation>) {
   );
 }
 
+function notNull(value: string | null): value is string {
+  return value !== null;
+}
+
 const filter = (values: (string | null)[]): string[] => {
-  const filtered = values.filter((v: string | null) => !!v);
+  const filtered = values.filter(notNull);
   const set = new Set(filtered);
-  const deduped: string[] = [];
-  set.forEach((value: string | null) => {
-    deduped.push(value!);
-  });
-  return deduped;
+  return Array.from(set);
```

Comment:
I simplified this construction and removed the type assertion on line 303 by using a type predicate (the `notNull` function and its `value is string` syntax).